elements/pfe-cta/src/pfe-cta.scss

```diff
@@ -25,9 +25,11 @@ $LOCAL-VARIABLES: (
   TextDecoration: none,
   TextDecoration--hover: none,
   TextDecoration--focus: none,
-  FontWeight: pfe-var(font-weight--semi-bold),
+  FontWeight: pfe-var(font-weight--bold),
   FontFamily: pfe-var(font-family--heading),
-  FontSize:   pfe-var(FontSize--md),
+  FontSize:   pfe-var(FontSize--lg),
+  FontSize--priority: pfe-var(FontSize--md), 
```

Comment:
I don't love these additions.  When we want to create a separate value for variant but it's the same CSS property, we just redefine the property and set a new fallback.  Otherwise we're going to massively bloat the local variable lists.

Reply:
the FontSize--priority already existed below. The only addition was FontSize--disabled. We could have the disabled button leverage the FontSize--priority var too, since it has a "priority" (button) style